packages/twenty-front/src/modules/auth/hooks/useLogout.ts

```diff
@@ -0,0 +1,15 @@
+import { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { useAuth } from '@/auth/hooks/useAuth';
+import { AppPath } from '@/types/AppPath';
+
+export const useLogout = () => {
```

Comment:
It might get confusing if we introduce a hook useLogout as it won't be clear for people what is the difference between useLogout and and signOut method of useAuth. Maybe we should be more explicit like `useSignOutAndExit`?
(alternative useSignOutAndRedirect, useSignOutWithRedirect, useSignOutWithExit)